tutorial/numpy-functions.md

```diff
@@ -329,14 +329,16 @@ In Pyccel we try to support the Numpy functions which developers use the most..
 
 -   Supported types : bool, int, int8, int16, int32, int64, float, float32, float64, complex64 and complex128. they can be used as cast functions too.
 
+    Note: for np.bool, np.int and np.float are just aliases to the Python native types, and these aliases are considered as a deprecated way to work with Python built-in types in NumPy.
```

Comment:
Thanks for adding this. Small corrections:
- The word `for` is not needed in this sentence, please remove it
- Avoid repetitions with this substitution: `these aliases` -> `they`

Reply:
Sentence improvement:
`np.bool, np.int and np.float are just aliases to the Python native types, and are a deprecated way to work with Python built-in types in NumPy.`